pkg/config/oracle_config.go

```diff
@@ -6,6 +6,7 @@ import "time"
 type OracleConfiguration struct {
 	Enabled               bool               `yaml:"Enabled"`
 	AllowPrivateHost      bool               `yaml:"AllowPrivateHost"`
+	AllowedContentTypes   []string           `yaml:"AllowedContentTypes"`
```

Comment:
We need our default configurations to contain this as well as `docs/oracle.md`.

Reply:
But what about testnet/mainnet? These are technically limited too and it's even more important than for private networks.